packages/web/src/scss/components/Toast/_Toast.scss

```diff
@@ -0,0 +1,122 @@
+// 1. Spread the `Toast` container over the viewport.
```

Comment:
Just passing by…
The first half of the file is comments, the rest is code. Don't see that very often. 
I wonder if you let Github Copilot generate code based on those comments. 😅

Reply:
Actually it's the other way round: I write the docs after having failed with writing various code variants.

Reply:
Now there is more comments than lines of code! 🎉